src/Akka.Streams.Kafka/Dsl/KafkaConsumer.cs

```diff
@@ -24,7 +25,7 @@ public static class KafkaConsumer
         /// </summary>
         public static Source<ConsumeResult<K, V>, Task> PlainSource<K, V>(ConsumerSettings<K, V> settings, ISubscription subscription)
         {
-            return Source.FromGraph(new KafkaSourceStage<K, V>(settings, subscription));
+            return Source.FromGraph(new PlainSourceStage<K, V>(settings, subscription));
```

Comment:
Some renaming to make it more similar to alpakka naming and thus making it simpler to compare/verify implementation.